modules/core/02-client/keeper/grpc_query.go

```diff
@@ -188,6 +188,46 @@ func (q Keeper) ConsensusStates(c context.Context, req *types.QueryConsensusStat
 	}, nil
 }
 
+// ConsensusStateHeights implements the Query/ConsensusStateHeights gRPC method
+func (q Keeper) ConsensusStateHeights(c context.Context, req *types.QueryConsensusStateHeightsRequest) (*types.QueryConsensusStateHeightsResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	if err := host.ClientIdentifierValidator(req.ClientId); err != nil {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	ctx := sdk.UnwrapSDKContext(c)
+
+	consensusStateHeights := []types.Height{}
```

Comment:
Nit: We tend to use code style of declare rather than initialise for slices like this, but it may not be consistent across the entire codebase. 

```suggestion
	var consensusStateHeights []types.Height
```

Reply:
thanks for this, I will note this so that my coding style is more align with you guys